include/openssl/pkcs12.h.in

```diff
@@ -45,7 +45,7 @@ extern "C" {
 
 # define PKCS12_MAC_KEY_LENGTH 20
 
-# define PKCS12_SALT_LEN 8
+# define PKCS12_SALT_LEN 16
```

Comment:
This is an API change. We cannot do this.

Reply:
This macro was only used in PKCS12_setup_mac. It is invalid for the application.
It is better move to inner headfile.

Reply:
Updated comments

Reply:
You need to change the value back to 8.

Reply:
The default salt length Changed back to 8, and add comments